R/pkg/R/mllib.R

```diff
@@ -142,15 +143,6 @@ setMethod("spark.glm", signature(data = "SparkDataFrame", formula = "formula"),
 #' Generalized Linear Models (R-compliant)
 #'
 #' Fits a generalized linear model, similarly to R's glm().
-#' @param formula A symbolic description of the model to be fitted. Currently only a few formula
```

Comment:
Well yes we agree on that. The reason I moved these is because I thought when `setGeneric("glm")` is called, it looks for the R glm method and it includes many arguments. Some overlap with `glm` defined for SparkDataFrame. Then I simply moved all the argument docs to the generic part, though that makes the doc there cumbersome...